### ISSUE TITLE
enable "features = ["cosmwasm_1_2"]"

### ISSUE THREAD

**Issue by taitruong** [CosmWasm/cw-multi-test#120]
Is there a reason, why `cosmwasm_1_2` explicitly needs to be enabled - shouldnt it be the default in `Cargo.toml`?

```toml
cw-multi-test = { version = "^0.20", features = ["cosmwasm_1_2"] }
```

**Comment by DariuszDepta** [comment]
Hi @taitruong,

Currently, developers have some freedom to select the proper cosmwasm dependency version, so anyone should set the proper version in `features`. When we define the default value, then users that need lower version will have to define it like this:
```
cw-multi-test = { version = "^0.20", default-features = false, features = ["cosmwasm_1_1"] }
```
Probably this will be ok for most users, especially because some older versions have reached EOL (see [this article](https://medium.com/cosmwasm/eol-for-cosmwasm-1-0-1-3-22df4b34b13c)).

Because version 1.2 has reached EOL with the end of 2023, shouldn't we then set `cosmwasm_1_3` as a default feature?
Should then version 1.3 be replaced with `cosmwasm_1_4` starting from the 1st of April 2024? We are currently preparing version 2.0 to be released. So maybe the default value should be `cosmwasm_2_0` soon?

As you see, there is no universal set of default dependencies - different users have different versions of CosmWasm, and they want to test them using **MultiTest**. So currently we decided NOT to specify default dependencies. At the cost of placing in Cargo.toml the one-liner you mentioned above.
